content/en/dev-ops/_index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "dev-ops"
+layout: "scenario"
+
+css: "scss/scenario.scss"
+
+section1:
+  title: KubeSphere DevOps offers end-to-end workflow and integrates popular CI/CD tools to boost delivery
+  content: KubeSphere provides rich observability from infrastructure to applications. It integrates your favorite tools for multi-dimensional monitoring metrics, multi-tenant log query and collection, alerting and notification.
+  image: /images/service-mesh/banner.jpg
+
+image: /images/dev-ops/dev-ops.jpg
+
+section2:
+  title: Automatically Checkout Code, Test, Analyse, Build, Deploy and Release
+  list:
+    - title: Multi-dimensional Monitoring
+      image: /images/observability/multi-dimensional-monitoring.png
+      contentList:
+        - content: <span>Easy to integrate with your SCM, </span> support GitLab / GitHub / BitBucket / SVN
+        - content: <span>Design a graphical editing panel</span> to create CI/CD pipelines, without writing Jenkinsfile
+        - content: <span>Integrate SonarQube</span> to implement source code quality analysis
+        - content: <span>Support dependency cache</span> to accelerate build and deployment
+        - content: <span>Provide dynamic build agents</span> to automatically spin up Pods as necessary
+    
+    - title: Built-in automated toolkits
+      image: /images/observability/log-query-and-collection.png
+      contentList:
+        - content: <span>Source to Image,</span> building reproducible container images from source code, without writing dockerfile
+        - content: <span>Binary-to-image</span>is the bridge between your artifact and a runnable image
+        - content: <span>Support automatically build and push</span> image to any registry, finally deploy to Kubernetes.
+        - content: <span>Good recoverability and flexibility,</span> you can rebuild and rerun S2I / B2I whenever a patch is needed
+    
+    - title: Use GitOps to implement DevOps, not just culture
+      image: /images/observability/flexible-alerting-and-notification.png
+      contentList:
+        - content: <span>Combine Git with Kubernetes’ convergence, and automates the cloud native Apps delivery</span>
+        - content: <span>Designed for teams, offer built-in multitenancy in DevOps project</span>
+        - content: <span>Liable to be observable,</span> provide dynamic logs for the S2I / B2I build and pipeline
+        - content: Provide audit, alert and notification in pipeline, ensuring quickly locate and solve issues.
+        - content: Support add Git SCM webhooks to trigger a Jenkins build when new commints to the branch
+
+section3:
+  title: See KubeSphere One-stop DevOps Workflow In Action
+  image: /images/service-mesh/15.jpg
```

Comment:
All the screenshots in devops page are not matched, you need to verify them with the design layout

![image](https://user-images.githubusercontent.com/40452856/84476959-77fdaa80-acc1-11ea-85d2-67d362b7fd1a.png)

Reply:
Those pictures can't be downloaded, I have already contacted the UI, temporarily use other pictures instead

Reply:
okay, our designer is on vacation today, I guess he might reply to you tomorrow.